nltk/translate/__init__.py

```diff
@@ -19,5 +19,6 @@
 from nltk.translate.ibm4 import IBMModel4
 from nltk.translate.ibm5 import IBMModel5
 from nltk.translate.bleu_score import bleu
```

Comment:
Possibly out of scope of this PR but BLEU should be considered in `nltk.translate.metrics` or maybe at some point `nltk.translate.metrics` will become a package by itself because there's a wide range of MT metrics in the literature.